src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -128,6 +141,22 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void CheckForUnsupportedArgs(Execution::Context& context)
+    {
+        const auto& unsupportedArgs = context.Get<Execution::Data::Installer>().value().UnsupportedArguments;
+
+        if (!unsupportedArgs.empty())

```

Comment:
I'm pretty sure you don't need to check for empty and can just let the `for` do its thing.

Reply:
Removed empty check